apps/server/src/modules/tool/controller/dto/response/oauth2-tool-config.response.ts

```diff
@@ -0,0 +1,37 @@
+import { ToolConfigType } from '@src/modules/tool/interface/tool-config-type.enum';
+import { ExternalToolConfigResponse } from '@src/modules/tool/controller/dto/response/external-tool-config.response';
```

Comment:
Please internel imports for example via ../interface or ./dto

Reply:
done